doc/en/announce/release-2.8.7.rst

```diff
@@ -0,0 +1,28 @@
+pytest-2.8.6
+============
+
+pytest is a mature Python testing tool with more than a 1100 tests
+against itself, passing on many different interpreters and platforms.
```

Comment:
I think it is worth mentioning it that such a short release was due to a serious regression